rplugin/python3/molten/jupyter_server_api.py

```diff
@@ -0,0 +1,134 @@
+import json
+import re
+import time
+import uuid
+from queue import Empty as EmptyQueueException
+from queue import Queue
+from threading import Thread
+from typing import Any, Dict
+from urllib.parse import urlparse
+
+import requests
+import websocket
```

Comment:
These are both packages that need to be installed, right? Can we put them inside the class/functions that they're used in to avoid erroring if people want to use the plugin without this feature and without installing these packages?

These requirements also need to be documented in the README